site/app/templates/autograding/TAResults.twig

```diff
@@ -77,37 +77,27 @@ $( document ).ready(function() {
-                                {% if file.encoded_name in annotated_file_names %}
-                                    {% if loop.index != uploaded_pdfs|length %}
-                                        <div style="margin-bottom: 2em">
-                                            {{ file.relative_name }}
-                                        </div>
-                                    {% else %}
-                                        {{ file.relative_name }}
-                                    {% endif %}
-                                {% endif %}
-                            {% endfor %}
                         </div>
-                        <div class="no-border-box col-md-4 offset-md-4" style="float: center; padding: 10px; width: 40%; word-break: break-word; display: inline-block">
-                            {% for file in uploaded_pdfs %}
-                                {% if file.encoded_name in annotated_file_names %}
-                                    <p style="text-align:center; vertical-align: middle; float: right; margin-bottom: 2em">
+                    </div>
+                    {% for file in uploaded_pdfs %}
+                        {% if file.encoded_name in annotated_file_names %}
+                            <div class="row">
+                                <div class="no-border-box col-md-4" style="float: right; word-break: break-word; display: inline-block">
+                                    <p>{{ file.relative_name }}</p>
+                                </div>
+                                <div class="no-border-box col-md-4 offset-md-4" style="float: center; word-break: break-word; display: inline-block">
+                                    <p style="text-align:center; vertical-align: middle; float: right">
                                         <a class="btn btn-success key_to_click" style="margin-right:20px;" tabindex="0" onclick="openUrl('{{ student_pdf_view_url }}?filename={{ file.name }}&path={{ file.path|url_encode }}&anon_path={{ file.anon_path|url_encode }}')">View Popup <i class="fas fa-window-restore"></i></a>
-                                        <button class = 'btn btn-default key_to_click' onclick='downloadStudentAnnotations("{{ student_pdf_download_url }}?filename={{ file.name }}&path={{ file.path|url_encode }}&anon_path={{ file.anon_path|url_encode }}", "{{file.path|url_encode}}", "annotated_pdfs")' aria-label="Download the file"> Download
+                                        <button class = 'btn btn-default key_to_click' style="margin-right:20px;" onclick='downloadStudentAnnotations("{{ student_pdf_download_url }}?filename={{ file.name }}&path={{ file.path|url_encode }}&anon_path={{ file.anon_path|url_encode }}", "{{file.path|url_encode}}", "annotated_pdfs")' aria-label="Download the file"> Download
                                         <i class="fas fa-download" title="Download the file"></i></button>
                                     </p>
```

Comment:
There's a bunch of inline CSS here, which could be improved by moving it to an external CSS stylesheet.